src/lib/zf/cplane.c

```diff
@@ -258,10 +274,10 @@ __zf_cplane_get_path(struct zf_stack* st, struct zf_path* path,
                       CI_MAC_PRINTF_ARGS(&data.dst_mac),
                       data.base.ifindex, data.encap.type);
 
-  if( iter > 0 ) {
+  if( wait ) {
     zf_log_cplane_info(st, "ARP for " CI_IP_PRINTF_FORMAT
                        " via ifindex %d took %dus\n",
-                       CI_IP_PRINTF_ARGS(&path->dst), data.base.ifindex, iter);
+                       CI_IP_PRINTF_ARGS(&path->dst), data.base.ifindex, time_used);
```

Comment:
Could you add some testing examples with this output just to show it all working with different attribute values.